src/main/java/racingcar/model/domain/Car.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.model.domain;
+
+public class Car {
+
+    private final static int NUMBER_ZERO = 0;
+    private final String name;
+    private int step;
+
+    public Car(final String name) {
+        this.name = name;
+        this.step = NUMBER_ZERO;
+    }
+
+    public void moveForward() {
+        this.step++;
+    }
+
+    public boolean isMax(final int max) {
+        return this.getStep() == max;
```

Comment:
굳이 메서드 호출을 통해 하지 않아도 될 것 같아요!

```suggestion
        return this.step == max;
```

Reply:
당연한건데 세세한 부분에서 자꾸 놓치네요 계속 연습하겠습니다 ㅠ